src/makeInfinityAdder.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let collection = 0;
+
+  const adder = (...nums) => {
```

Comment:
Adder function should take only 1 argument.

![image](https://github.com/mate-academy/js_task-inifinite-sum-closures/assets/84296041/7447d60a-bafd-4c37-9d2e-f0ee2c089210)

```suggestion
  const adder = (number) => {
```